sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -58,7 +58,7 @@ your resource and a `credential` that allows you access:
 ```python
 from azure.ai.textanalytics import TextAnalyticsClient
 
-text_analytics = TextAnalyticsClient(endpoint="https://westus2.api.cognitive.microsoft.com/", credential=credential)
+text_analytics_client  = TextAnalyticsClient(endpoint="https://westus2.api.cognitive.microsoft.com/", credential=credential)
```

Comment:
not sure what the guideline is for Python, but could the endpoint be a variable that the user can populate?

Reply:
Do you mean use a variable to shorten the length of the line? Or are you asking if the user can pass the endpoint in as a variable instead of a keyword argument?

Reply:
something like `TextAnalyticsClient(endpoint=endpoint, credential=credential)`
otherwise sounds like they have to use the endpoint you are providing

Reply:
I'm on the fence about this one. Past ux studies have shown that users did not know what an endpoint looked like and so we started including an example like this (e.g. storage). If I removed the region do you think it would be more clear that this isn't the endpoint to use: 

`https://<region>.api.cognitive.microsoft.com/`

Reply:
thanks for the context. Let's keep it in accordance to what UX Study have found. So no change then.